src/main/kotlin/blackjack/domain/player/Player.kt

```diff
@@ -0,0 +1,39 @@
+package blackjack.domain.player
+
+import blackjack.domain.card.Card
+import blackjack.domain.card.CardNumber
+
+class Player private constructor(
+    val name: String,
+    private var cards: List<Card>
+) {
```

Comment:
```suggestion
class Player private constructor(
    val name: String,
    vararg cards: Card,
) {
    private val cards: MutableList<Card> = cards.toMutableList()

// ...

companion object {
        fun from(name: String, firstCard: Card, secondCard: Card) = Player(name, firstCard, secondCard)
    }
```

- 가변인자(`vararg`)를 활용해볼 수도 있겠네요.
- `cards` 프로퍼티는 `MutableList` 를 사용해서 variable 대신 value 로 사용할 수 있을거 같아요. 이러면 기존과 어떤 차이가 생길까요?

Reply:
> 가변인자(vararg)를 활용해볼 수도 있겠네요.

이거 기가막히네요... 제가 찾던 게 바로 이건데.. 대박적,,,
var 말고 val 를 쓰면 최초의 mutableList 안에서만 값 수정이 되고 mutableList 자체의 주소값은 바꿀 수 없는 거 ,,, 맞나요!??!?!

Reply:
리스트 자체는 재할당할 수 없고, 리스트 내부 원소들만 CRUD 가 가능해진다!